style.css

```diff
@@ -7,11 +7,20 @@
   list-style: none;
 }
 
```

Comment:
- [ ] [OPTIONAL] Wow, everything is working fine 👏, but here is something left. Please kindly delete lines between `640` and `642`. I am sure you can fix it. Please see the images below to see the difference in your project 😊🍀
```css
  .green-btn {
    margin-left: -173%;
  }
```
**Your current design**

![image](https://user-images.githubusercontent.com/69011963/146360779-349f44cf-5af8-4369-9e2c-0bf9486cd092.png)

**After removed styles**

![image](https://user-images.githubusercontent.com/69011963/146360934-e36ed212-0c1b-44ac-bfd9-7b73d2aff9a0.png)